autogluon/utils/tabular/ml/trainer/model_presets/presets.py

```diff
@@ -87,6 +89,10 @@ def get_preset_models_classification(path, problem_type, objective_func, stoppin
             TabularNeuralNetModel(path=path, name='NeuralNetClassifier', problem_type=problem_type,
                                   objective_func=objective_func, stopping_metric=stopping_metric, hyperparameters=nn_options.copy()),
         )
+    if lr_options is not None:
+        models.append(
+            LinearModelNoTextFeatures(path=path, name='LinearModelNoTextFeatures', problem_type=problem_type, objective_func=objective_func, hyperparameters=lr_options.copy())
```

Comment:
Why is this not `LinearModel` as the default?

Moreover, should the 'LR' hyperparams dict include a key 'handle_text' with options: 'include' (use `LinearModel`), 'ignore' (use `LinearModelNoText`), 'only' (use `LinearModelOnlyText`)?

Otherwise it seems the user cannot easily access these without delving into undocumented functionalities like Trainer.

Reply:
* Switched to single `LinearModel` class with `handle_text` hyperparameter option.